[IMP] inventory: purchase security lead time (#9178)
odoo/documentation: content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates.rst

   :align: center
   :alt: Show graphic of all lead times working together.

- :ref:`Customer lead time <inventory/shipping_receiving/customer-lt>`: The default time frame for
  fulfilling customer orders. The customer lead time is the number of days from the date the sales
  order (SO) is confirmed to the date the products are shipped from the warehouse. This is also
  known as *delivery lead time*.

- :ref:`Sales security lead time <inventory/shipping_receiving/sales-security-lt>`: moves the
  *scheduled delivery date* forward by a specified number of days. This serves as a buffer to allow

>   the team ample time to prepare the outgoing shipment earlier, considering the possibility of

Reviewer: ```suggestion
   ample time to prepare the outgoing shipment earlier, considering the possibility of
```
Optional - "the team" sounded a little off/vague to me

  delays in the fulfillment process.

- :ref:`Purchase lead time <inventory/shipping_receiving/purchase-lt>`: the number of days from the
  confirmation of a purchase order (PO) to the receipt of products. It provides insight on the time
  it takes for products to arrive at the warehouse, facilitating effective scheduling and planning
  of supplier deliveries.

- :ref:`Purchase security lead time <inventory/shipping_receiving/purchase-security-lt>`: advances

>   the order deadline on a :abbr:`PO (Purchase Order)` by a specified number of days. This proactive

Reviewer: If you want you can also do a replace at the top for PO, like you did the other abbr tags

  approach of placing orders earlier mitigates the risk of vendor or shipping delays. Thus, for
  products that are set to replenish to order, the need appears on the *Replenishment report*
  earlier, according to the specified number of days.

- :ref:`Days to Purchase <inventory/shipping_receiving/days-to-purchase>`: days needed for the
  vendor to receive a request for quotation (RFQ) and confirm it. It advances the deadline to
  schedule a |RFQ| by a specified number of days.

> - :ref:`Manufacturing lead time <inventory/shipping_receiving/manuf-lt>`: the number of days needed

Reviewer 2: ```suggestion
- :ref:`Manufacturing lead time <inventory/shipping_receiving/manuf-lt>`: number of days needed
```

Manufacturing lead time
-----------------------

> Manufacturing lead times for products are configured from a product's Bill of Materials (BoM) form.

Reviewer: ```suggestion
Manufacturing lead times for products are configured from a product's bill of materials (BoM) form.
```

consider weekends, holidays, or *work center capacity* (:dfn:`the number of operations that can be
performed at the work center simultaneously`).

.. seealso::
   - :doc:`Manufacturing planning <../../../manufacturing/workflows/use_mps>`
   - :doc:`Schedule MOs with reordering rules
     <../../product_management/product_replenishment/reordering_rules>`

.. example::
   A product's scheduled shipment date on the :abbr:`DO (Delivery Order)` is August 15th. The
   product requires 14 days to manufacture. So, the latest date to start the :abbr:`MO
   (Manufacturing Order)` to meet the commitment date is August 1st.

.. _inventory/shipping_receiving/prepare-manufacturing-order:

Days to prepare manufacturing order
-----------------------------------

Configure the days required to gather components to manufacture a product by going to its |BoM|. To
do that, go to :menuselection:`Manufacturing app --> Products --> Bills of Materials`, and select
the desired |BoM|.

In the :guilabel:`Miscellaneous` tab of the |BoM|, specify the calendar days needed to obtain
components of the product in the :guilabel:`Days to prepare Manufacturing Order` field. Doing so

> creates |MOs| in advance, ensure there is enough time to replenish components, or manufacture

Reviewer: ```suggestion
creates |MOs| in advance, and ensures there is enough time to either replenish components, or manufacture
```
